slash/core/session.py

```diff
@@ -93,6 +93,12 @@ def deactivate(self):
         self._logging_context = None
         ctx.pop_context()
 
+    def get_current_duration(self):
```

Comment:
@ayalash I prefer a single way of getting the duration. Perhaps make `duration` a property that will support both cases with the use of `start_time` and `end_time`?